recipes/base_install.rb

```diff
@@ -26,12 +26,18 @@
     include_recipe "yum-epel"
   end
 
-
   unless node['platform_version'].to_i < 7
     execute 'yum-config-manager_skip_if_unavail' do
       command "yum-config-manager --setopt=\*.skip_if_unavailable=1 --save"
```

Comment:
should we set this flag for dnf too?

Reply:
This command actually "just works" on CentOS 8, and should be equivalent to running `dnf config-manager`.
We could change it if we want but there should be no difference